test/pkg/integration/setup.go

```diff
@@ -143,68 +144,35 @@ func Setup(ctx context.Context) (string, string, env.Environment, bool, string,
 	return username, e.Namespace(), testenv, enterprise, kubeconfig, gitlab
 }
 
+var (
+	components = []string{
```

Comment:
Some (new) components are not in this list, like `node-labeler`, which is probably okay, because they aren't in the hot path.

Are there any other newer components, that should be in this list? From webapp, like redis or spideDB? From IDE, like `ide-service`?

Reply:
yeah good point, let me update the list

Reply:
Updated, also changed the logic to check all components for readiness to list all pods once from the k8s API, instead of listing for each component separately. Because of the increase of components, the latter was starting to log client-side k8s client throttling warnings.

Reply:
Awesome, thanks @WVerlaek ! Should we reduce how often we poll in [here](https://github.com/gitpod-io/gitpod/pull/18049/files#diff-b03ae37a74c87aef701e0b1cbcf747a9572a3a1cae7706c54722ccb126286cbbR189), perhaps to every 5 or 10s?

Reply:
Not really needed now, as with this change we only do one List call to get all pods when checking the preview env's readiness